src/Cli/Configuration/DomainConfiguration.php

```diff
@@ -48,7 +48,23 @@ public function getConfigTreeBuilder()
                         ->ifTrue(function ($item) {
                             return !filter_var($item, FILTER_VALIDATE_EMAIL);
                         })
-                        ->thenInvalid('The email "%s" is not valid.')
+                        ->thenInvalid('The email %s is not valid.')
+                    ->end()
+                ->end()
+                ->scalarNode('key_type')
+                    ->info('Type of private key (RSA or EC).')
+                    ->defaultValue('RSA')
```

Comment:
I guess the default should be in lowercase `rsa`?

Reply:
normalized to lower case. but maybe I should normalized everything to uppercase...